manifests/client_specific_config.pp

```diff
@@ -12,6 +12,7 @@
 # @param dhcp_options DHCP options to push to the client.
 # @param redirect_gateway Redirect all traffic to gateway
 # @param ensure Sets the client specific configuration file status (present or absent)
+# @param manage_client_configs Manage dependencies on Openvpn::Client ressources
```

Comment:
The description is Puppet focused.
A description more openvpn focused looks clearer IMO.

Reply:
I've written it like this because it doesn't really do anything apart from adding the dependency. Having said that, I just looked at the client resource and I saw that it also defines a similar ordering constraint: https://github.com/voxpupuli/puppet-openvpn/blob/master/manifests/client.pp#L89

When I look at the client_specific_config resource, I see that it only adds one file. The directory of that file is created by openvpn::server. Shouldn't it be enough to order `server -> client_specific_config` then? I don't see why the client resources should be ordered before client_specific_config.

If this sounds right to you, I'd simplify my patch by removing the boolean that I introduced and then removing the ordering constraint `server->client` from client_specific_config altogether.

Reply:
Since we do not have acceptance tests that covert this, i feel more confident to keep your patch like this.

Or, we have to add acceptance tests that covert modification of client_specific_config.

Up to you.

Reply:
Keeping it as is is fine with me.

Reply:
@Bluewind thank you for the PR.